app/controllers/hub/state_routings_controller.rb

```diff
@@ -43,12 +43,12 @@ def state_routing_params
 
     def load_state_and_routing_targets
       @state = params[:state]
-      @coalition_srts = StateRoutingTarget.where(state_abbreviation: @state, target_type: Coalition.name).includes(:state_routing_fractions)
-      @independent_org_srts = StateRoutingTarget.where(state_abbreviation: @state, target_type: VitaPartner.name).includes(:state_routing_fractions)
+      @coalition_srts = StateRoutingTarget.includes(:target, state_routing_fractions: :vita_partner).where(state_abbreviation: @state, target_type: Coalition.name)
+      @independent_org_srts = StateRoutingTarget.includes(target: :child_sites, state_routing_fractions: :vita_partner).where(state_abbreviation: @state, target_type: VitaPartner.name)
     end
 
     def load_independent_organizations
-      @independent_organizations = Organization.where(coalition: nil).where.not(id: @independent_org_srts.pluck(:target_id))
+      @independent_organizations = Organization.where(coalition: nil).where.not(id: @independent_org_srts.map(&:target_id))
```

Comment:
can you explain why this change from pluck to map? Is `pluck` making a new sql query whereas we already have the `@independent_org_srts` at this point (already fetched from db) so we want to avoid using `pluck` and instead iterate over the objects already in-memory?